hail/src/main/scala/is/hail/utils/richUtils/RichCodeRegion.scala

```diff
@@ -4,9 +4,11 @@ import is.hail.annotations.Region
 import is.hail.asm4s._
 
 class RichCodeRegion(val region: Code[Region]) extends AnyVal {
-  def allocate(alignment: Code[Long], n: Code[Long]): Code[Long] = {
+  def allocate(n: Code[Long]): Code[Long] =
```

Comment:
Where is this used?

Reply:
Looks like this was left from an earlier version. Will remove.